### PR TITLE
Feat docker julia

### PR DESCRIPTION
**Pull request by tdivoll**
This PR builds a Docker image and workflow for Julia and the cli scripts called in the Cylc pipeline

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+RUN apt-get clean && apt-get update && \
+apt-get install -y git
+
+RUN cd opt 
+
+WORKDIR /opt
+
+RUN git clone https://github.com/WilhelmusLab/ice-floe-tracker-pipeline.git && \ 
+git clone https://github.com/WilhelmusLab/IceFloeTracker.jl.git IceFloeTracker
```
Comment (cpaniaguam) [inline Dockerfile:13]:
```suggestion
git clone https://github.com/WilhelmusLab/IceFloeTracker.jl.git
```

Comment (tdivoll) [reply Dockerfile:13]: This just gives the repo a simpler name in the environment, so users aren't thinking that `IceFloeTracker.jl` is a script. @mirestrepo pointed out that it looks like a script in the build commands, which is a bit confusing.

Comment (cpaniaguam) [reply Dockerfile:13]: Cool, didn't know that trick!

**Comment by tdivoll** [comment]
This one is ready for review. For now, it has fetchdata and soit pulling software deps from the icefloetracker-fetchdata container, and all the Julia functions run as Cycl tasks right from the repo rather than a second Julia container. 

For now I left the Julia container build in there in case we decide it is needed. We could delete that on a different PR after consulting the larger group.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+RUN cd opt 
+
+WORKDIR /opt
+
+#RUN pip3 install -U scikit-image==0.20.0 pyproj==3.6.0 rasterio==1.3.7 requests==2.31.0 skyfield==1.45.0
+
+RUN git clone https://github.com/WilhelmusLab/ice-floe-tracker-pipeline.git
+
+#RUN julia -e 'using Pkg; Pkg.activate("/opt/IceFloeTracker"); ENV["PYTHON"]=""; Pkg.instantiate(); Pkg.build("PyCall")'
```
Comment (cpaniaguam) [inline Dockerfile:17]:
```suggestion
RUN git clone https://github.com/WilhelmusLab/ice-floe-tracker-pipeline.git
```
Remove the comments?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+RUN julia -e 'using Pkg; Pkg.activate("/opt/ice-floe-tracker-pipeline"); Pkg.instantiate()'
+
+RUN chmod a+x /opt/ice-floe-tracker-pipeline/workflow/scripts/ice-floe-tracker.jl
+
+CMD [ "/bin/bash", "-c" ]
```
Comment (cpaniaguam) [inline Dockerfile:23]:
```suggestion
CMD [ "/bin/bash", "-c" ]

```

File: README.md [hunk 4]
```diff
@@ -30,39 +30,34 @@ The text-based user interface provides a simple way to watch the status of each
...
 
-7. Export SOIT username/password to environment variable
+6. Export SOIT username/password to environment variable
    - [ ] From your home directory`nano .bash_profile`
    - [ ] add `export HISTCONTROL=ignoreboth` to the bottom of your .bash_profile
         * this will ensure that your username/password are not stored in history
-        * when exporting the following environment variables, there must a space in front of each command
+        * when exporting the following environment variables, there __must__ be a space in front of each command
    - [ ] ` export SPACEUSER=<firstname>_<lastname>@brown.edu`
    - [ ] ` export SPACEPSWD=<password>`
```
Comment (cpaniaguam) [inline README.md:58]: Why not provide the user with a file to populate with the credentials that only the user can read (`chmod 400 file-with-credentials`)? I just think it's very easy to forget to type that space!

**Review by cpaniaguam — APPROVED** [review]
I think this is good!

However, the space thing for not logging commands seems a bit fragile. There are also a couple of commented lines that could be deleted.

File: README.md (hunk 4 above)
Comment (tdivoll) [reply README.md:58]: I think it is better to only have that stored in RAM during the session. Having a file means it persists on the disk and then could potentially be hacked.

**Pull request merged**